mobile/ios/LightningApp.xcodeproj/xcshareddata/xcschemes/LightningApp.xcscheme

```diff
@@ -99,6 +99,13 @@
             ReferencedContainer = "container:LightningApp.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "OS_ACTIVITY_MODE"
+            value = "${DEBUG_ACTIVITY_MODE}"
+            isEnabled = "NO">
+         </EnvironmentVariable>
+      </EnvironmentVariables>
```

Comment:
Couldn't see in the docs where this is set?

Reply:
Oops! This was meant to be taken out.